jcommune-service/src/main/java/org/jtalks/jcommune/service/nontransactional/BBCodeService.java

```diff
@@ -73,13 +74,21 @@ public String convertBbToHtml(String bbEncodedText) {
         for (TextProcessor preprocessor : preprocessors) {
             bbEncodedText = preprocessor.process(bbEncodedText);
         }
-        bbEncodedText = processor.process(bbEncodedText);
+        if (isUserbbCode(bbEncodedText)){
```

Comment:
I'm not sure I fully understand the changes - why do we process text differently depending on whether `[user]` bb code is present?